lib/agent0/agent0/hyperdrive/policies/lpandarb.py

```diff
@@ -399,14 +355,7 @@ def calc_reserves_to_hit_target_rate(
     while float(abs(predicted_rate - target_rate)) > TOLERANCE:
         iteration += 1
         latest_fixed_rate = calc_apr_local_dict(pool_config, pool_info)
-        target_bonds = calc_bond_reserves(
-            pool_info["shareReserves"],
-            pool_info["shareAdjustment"],
-            pool_config["initialSharePrice"],
-            target_rate,
-            pool_config["positionDuration"],
-            pool_config["invTimeStretch"],
-        )
+        target_bonds = interface.bonds_given_rate_and_shares(target_rate)
```

Comment:
this doesn't work with a changing `pool_info` object, right?

Reply:
That's a good point -- I'm going to put it back to using your function. We can revisit after #1042 merges. I think after that it'll be an easy fix to incorporate optional modified pool info or current pool info.